### DISCUSSION TITLE
Mismatch in GPG keys is confusing

### DISCUSSION THREAD

**Discussion by RyanZim** [evilsocket/opensnitch#1050] [Releases]
Recent releases state in the release notes:

> Packages signed with the following GPG key: [F34016AC014BAAF8C90AC730141D0D4E9FF44A67](https://keyserver.ubuntu.com/pks/lookup?search=F34016AC014BAAF8C90AC730141D0D4E9FF44A67&fingerprint=on&op=index)

However, the release assets include a `gustavo_iniguez_goia.asc` file, which contains the key 6F357A6209EB5C72005315C447F14912BCF6BE9C instead, which is confusing. 

It looks like usage of the F34016AC014BAAF8C90AC730141D0D4E9FF44A67 key started with [v1.6.0-rc.5](https://github.com/evilsocket/opensnitch/releases/tag/v1.6.0-rc.5), perhaps this is just an oversight, and the key in the assets should be updated?

**Comment by gustavo-iniguez-goya** [comment]
Hi @RyanZim ,

Could you try it with this one?  
[gustavo_iniguez_goia-9FF44A67.asc.txt](https://github.com/evilsocket/opensnitch/files/12809197/gustavo_iniguez_goia-9FF44A67.asc.txt)

**Reply by RyanZim** [reply]
Yeah, that works. I was able to pull the key from the keyserver as well; just thought I'd point out the mismatch; not sure if it was intentional.

**Reply by gustavo-iniguez-goya** [reply]
no, I uploaded my previous pub key :-1: I'll post it again.
